omnibus/config/software/python3.rb

```diff
@@ -12,12 +12,8 @@
   dependency "liblzma"
   dependency "libyaml"
 
-  version "3.6.7" do
-    source :sha256 => "b7c36f7ed8f7143b2c46153b7332db2227669f583ea0cce753facf549d1a4239"
-  end
-
-  version "3.7.1" do
-    source :sha256 => "36c1b81ac29d0f8341f727ef40864d99d8206897be96be73dc34d4739c9c9f06"
+  version "3.7.4" do
+    source :sha256 => "d63e63e14e6d29e17490abbe6f7d17afb3db182dbd801229f14e55f4157c4ba3"
```

Comment:
if we have only 1 version can we remove the `version XXX do [...] end` ?

Reply:
makes sense, done 👍